### PR TITLE
static api: adds support for 'scale' parameter

### PR DESCRIPTION
**Pull request by jacoscaz**
Hello!

There's an undocumented `scale` parameter that allows for @2x icons in markers. I've simply added it to the array of supported parameters in `lib/utils/parseMarkers.js`. If you're happy with this I can add a little paragraph about it in the readme.

Source:  http://stackoverflow.com/questions/10336646/how-can-i-use-high-resolution-custom-markers-with-the-scale-parameter-in-google/17130379#17130379

### CODE REVIEW

**Comment by fabriziomoscon** [comment]
Sounds good! Go for it.

**Comment by jacoscaz** [comment]
@fabriziomoscon done! Feel free to reword it to your liking.

**Comment by jacoscaz** [comment]
@fabriziomoscon I've rebased my fork to tidy things up as I caught a few wording mistakes. Should be ok now.
